ssl/ssl_init.c

```diff
@@ -105,5 +134,7 @@ int OPENSSL_init_ssl(uint64_t opts, const OPENSSL_INIT_SETTINGS *settings)
         && !RUN_ONCE(&ssl_strings, ossl_init_load_ssl_strings))
         return 0;
 
+    pin_library();
```

Comment:
OPENSSL_init_ssl() can be called multiple times, so this call should be a RUN_ONCE() or simply called from ossl_init_ssl_base()?